docs/specs/toc.yml

```diff
@@ -439,7 +439,7 @@ outputs:
   docs/toc.json: |
     {
       "items": [
-        { "name": "Azure", "href": "/azure/", "tocHref": "/azure/", "topicHref": "/azure/index", "homepage": "/azure/index" }
+        { "name": "Azure", "href": "/azure/", "tocHref": undefined, "topicHref": undefined, "homepage": "/azure/index" }
```

Comment:
`tocHref` and `topicHref` are not part of the output:

https://dev.azure.com/ceapex/Engineering/_git/docs-ui?path=%2Fpackages%2Fscripts%2Fsrc%2Ftoc%2Ftypes.ts